docs/src/main/asciidoc/mongodb-panache.adoc

```diff
@@ -513,6 +513,10 @@ Here are some query examples:
 - `lastname is not null` will be mapped to `{'lastname':{'$exists': true}}`
 - `status in ?1` will be mapped to `{'status':{$in: [?1]}}`
 
+WARNING: MongoDB queries must be valid JSON documents,
+using multiple times the same field on a query is not allowed using PanacheQL as is it would generate an invalid JSON
+(see link:https://github.com/quarkusio/quarkus/issues/12086[this issue on github]).
```

Comment:
Can we avoid referencing an issue? Better put an example in the doc of what's not valid.

Btw, I think it's a pretty annoying limitation given it can be a very common use case.

Reply:
> Btw, I think it's a pretty annoying limitation given it can be a very common use case.

That's why there is an open issue about it, so people can give feedback on it and maybe work on it. I think it's important to link to the openned issue as we could gather feedback about this issue and decide if we need to work on it (this would not be easy).

Reply:
I can off course add an example but I think the issue still needs to be linked here.

Reply:
@gsmet what your takes on my previous comment ?